ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh

```diff
@@ -5,10 +5,14 @@ set -o pipefail
 
 function extract_oc(){
     echo -e "Extracting oc\n"
-    local retry=5 
+    local minor_version retry=5 binary='oc'
     tmp_oc="/tmp/client"
     mkdir -p ${tmp_oc}
-    while ! (env "NO_PROXY=*" "no_proxy=*" oc adm release extract -a "${CLUSTER_PROFILE_DIR}/pull-secret" --command=oc --to=${tmp_oc} ${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE});
+    minor_version="$(oc get clusterversion --no-headers | awk '{print $2}' | cut -f2 -d.)"
```

Comment:
We are supposed to use `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` to get the value of `minor_version`, right?

Reply:
yes, `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` is the expected one for preupgrade/postupgrade.